source/include/core_sntp_client.h

```diff
@@ -40,6 +40,41 @@
 /* Include coreSNTP Serializer header. */
 #include "core_sntp_serializer.h"
 
+/* SNTP_DO_NOT_USE_CUSTOM_CONFIG allows building the SNTP library
+ * without a custom config. If a custom config is provided, the
+ * SNTP_DO_NOT_USE_CUSTOM_CONFIG macro should not be defined. */
+#ifndef SNTP_DO_NOT_USE_CUSTOM_CONFIG
+    /* Include custom config file before other headers. */
+    #include "core_sntp_config.h"
+#endif
+
+/* Include config defaults header to get default values of configs not
+ * defined in core_sntp_config.h file. */
+#include "core_sntp_config_defaults.h"
+
+/**
+ * @brief The default UDP port supported by SNTP/NTP servers for client-server
+ * communication.
+ *
+ * @note It is possible for a server to use a different port number than
+ * the default port when using the Network Time Security protocol as the security
+ * mechanism for SNTP communication. For more information, refer to Section 4.1.8
+ * of [RFC 8915](https://tools.ietf.org/html/rfc8915).
+ */
+#define SNTP_DEFAULT_SERVER_PORT    ( 123U )
+
+/**
+ * @brief core_sntp_struct_types
```

Comment:
I don't think this block is meant to have two `@brief`s
```suggestion
 * @ingroup core_sntp_struct_types
```